lux/test/unit/lux/lenses/etherscan/accounts/balance_lens_test.exs

```diff
@@ -0,0 +1,162 @@
+defmodule Lux.Lenses.Etherscan.BalanceLensTest do
+  use UnitAPICase, async: false
+
+  alias Lux.Lenses.Etherscan.BalanceLens
+
+  setup do
+    # Set up test API key in the configuration
+    Application.put_env(:lux, :api_keys, [
```

Comment:
The comment about application.put_env from earlier is true for all these tests...